Makefile

```
@@ -9,10 +9,13 @@ REPO_URL="https://github.com/opendatahub-io/odh-manifests"
 KFDEF_FILE="kfdef/kfctl_openshift.yaml"
 COMMIT_MESSAGE="Update KFdef for release $(TAG_NAME)"
 UPDATE_TO_COMMIT=master
+KUBE_LINTER_CONFIG=.kube-linter.yaml
 ifndef VERSION
 $(help)
 endif
 
+.PHONY: lint
```

Can you change this to `kube-lint`?

```suggestion
.PHONY: kube-lint
```